packages/flutter_blue/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: flutter_blue_tizen
+description: A new flutter plugin project.
+version: 0.0.1
+homepage:
```

Comment:
https://github.com/flutter-tizen/flutter-tizen/wiki/New-plugin-submission-guideline#pubspecyaml

Please fill the information properly.